src/mixins/state.ts

```diff
@@ -74,6 +75,22 @@ export default class StateMixin extends Vue {
     return this.printerState.toLowerCase() === 'printing'
   }
 
+  get printerPoweredOff (): boolean {
+    if (this.klippyConnected) {
+      return false
+    }
```

Comment:
This doesn't seem to work for me - Moonraker always reports Klippy as connected (but shut down):
![image](https://github.com/user-attachments/assets/414e079e-18dd-431c-86a1-4e611da6b518)

Changing this to `this.klippyReady` seems to work better, but will require the first two `<template>`s in SystemControl to be swapped (first the `<template v-if="printerPoweredOff">"`, then the `<template v-else-if="klippyConnected">`)

Reply:
Maybe I am missing something, but if `klippy_connected: true`, then it means the Moonraker is in effect connected to Klipper, so the printer has power (this method is to check if printer isn't powered on, hence we return `false` as it IS powered on)

Reply:
Yep, I would interpret it in the same way, but for whatever reason Moonraker does report Klippy to be connected (though the printer is definitely powered off)..

Reply:
I wonder if it is reporting the host klippy connection 🤔

Reply:
Did you add `bound_services: klipper` to the power element?

```ini
[power printer]
type: <any type>
...
bound_services: klipper
```

This MUST be set as it will stop the service when the power is off and turn the service back on when power is on (also matches [this](https://docs.mainsail.xyz/overview/quicktips/printer-power-switch))

Reply:
ah, this must be why! i didn't have this set, i'll test your changes again when i get home ;)

Reply:
I will add a page to the docs on this topic!

Reply:
Yep, adding the bound service fixed it!